projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -467,38 +566,112 @@ const isStepEligibleToShow = ( state, step ) => {
 			return isConditionalRecommendationEnabled( state, step ) && ! isFeatureActive( state, step );
 		case 'boost':
 			return isConditionalRecommendationEnabled( state, step ) && ! isFeatureActive( state, step );
+		case 'welcome__complete':
+		case 'welcome__security':
+		case 'welcome__antispam':
+		case 'welcome__videopress':
+		case 'welcome__search':
+		case 'welcome__scan':
+		case 'backup-activated':
+		case 'scan-activated':
+		case 'antispam-activated':
+		case 'videopress-activated':
+		case 'search-activated':
+		case 'server-credentials':
+			return true; // Can we check if credentials were added?
```

Comment:
I am not sure about that, you may want to ask one of the relevant slack channels about this to see if there is. Someone will know 😄